sdk/core/core-util/src/checkEnvironment.ts

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
also delete the `/src/isNode.ts`?

Reply:
Yes, the intent, if approved, would be to delete `isNode` and replace with this.